package-lock.json

```diff
@@ -0,0 +1,16946 @@
+{
```

Comment:
No more needed since #18.

Reply:
hey @antleblanc , I've tried `git checkout master package-lock.json` and i'ts not doing the trick.
I don't want create a new branch& PR just for that... Do you know how I can solve that ?